public/static/docs/command-reference/get.md

```diff
@@ -140,11 +140,12 @@ https://remote.dvc.org/get-started/66/2eb7f64216d9c2c1088d0a5e2c6951
 
 ## Example: Compare different versions of data or model
 
-`dvc get` provides the `--rev` option to specify which version of the repository
-to download a <abbr>data artifact</abbr> from. It also has the `--out` option to
-specify the location to place the artifact within the workspace. Combining these
-two options allows us to do something we can't achieve with the regular
-`git checkout` + `dvc checkout` process – see for example the
+`dvc get` provides the `--rev` option to specify which
+[Git revision](https://git-scm.com/docs/revisions) of the repository to download
```

Comment:
which commit ?

Reply:
OK. Should I just change all "revision" words to "commit" as much as possible even when talking about `rev` field or `--rev` options? And should the link stay, be removed, or changed to https://git-scm.com/docs/gitglossary#def_commit or https://git-scm.com/docs/git-commit ?

Reply:
probably not ... in most examples yes ... in other place we can do `commit ((revision)[link])` ... so not one rule fits all :)

Reply:
Yeah that's what I meant by "as much as possible" but let's leave this update for when the core repo stops using "revision" as much (in favor or "commit"). I just change this one instance (in a1c782a0).